apiritif/loadgen.py

```diff
@@ -176,9 +176,12 @@ def run_nose(self, params):
             config.stream = open(os.devnull, "w")  # FIXME: use "with", allow writing to file/log
         try:
             while True:
+                iteration += 1
+
+                log.info("Starting iteration:: index=%d,start_time=%.3f", iteration, time.time())
                 ApiritifTestProgram(config=config)
+                log.info("Finishing iteration:: index=%d,end_time=%.3f", iteration, time.time())
```

Comment:
`::` - is it intentional

Reply:
It is, though not very pretty. Needed some kind of separator that is easily detectable from a script and simple `:` wasn't fitting.

Will think of a way to improve this.